packages/core/src/account.ts

```diff
@@ -227,6 +227,25 @@ export class Account extends PBES2Container implements Storable {
         this._key = account._key;
     }
 
+    validateOrThrowInputValues() {
```

Comment:
`Serializable` actually already has a [`validate`](https://github.com/padloc/padloc/blob/v4/packages/core/src/encoding.ts#L170-L172) method that is automatically called when objects are deserialized. Might as well use that 😉